docs/migration-guides/prometheus-migration.md

```diff
@@ -347,3 +355,25 @@ level=info ts=2018-12-05T11:00:09.952061917Z caller=head.go:446 component=tsdb m
 ```
 
 It is good practice to leave the old Prometheus database running until you are satisfied that the migration has completed without errors.
+
+## Step 6: Clean up sidecar container
+
+Once you finished the migration, it's time to cleaning up the sidecar containers we added to help us with the TSDB migration. To do so, remove the block related to the rsync container:
+
+```jsonnet
+-                rsync: kube.Container("sidecar-rsync") {
```

Comment:
This doesn't match the earlier example.  I think it should be:

```patch
   containers_: containers_obj(super.containers) {
-    [rsync_container_name]: rsync_container,
     [prometheus_container_name]+: {
```

Alternatively you could just say something about re-pushing with the original unmodified config, which might be simpler (and a lot more robust).

Reply:
> Alternatively you could just say something about re-pushing with the original unmodified config, which might be simpler (and a lot more robust).

+1